spring-cloud-gcp-examples/spring-cloud-gcp-sql-example/src/main/java/com/example/WebController.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright 2017 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.example;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
```

Comment:
Don't know "why?", but Travis doesn't like this:
```
/home/travis/build/spring-cloud/spring-cloud-gcp/spring-cloud-gcp-examples/spring-cloud-gcp-sql-example/src/main/java/com/example/WebController.java:24: error: Wrong order for 'java.util.List' import.
```

Reply:
I'd forgotten to organize imports before..